src/actions/catActions.js

```diff
@@ -0,0 +1,31 @@
+
+import * as types from './actionTypes';
+import axios from 'axios';
+
+export const giphy_instance = axios.create({
+    "baseURL": "http://api.giphy.com/v1/gifs/",
+});
+
+// Alter defaults after instance has been created
+delete giphy_instance.defaults.headers.common.Authorization;
```

Comment:
Is this necessary? It shouldn't have any authorization headers set when creating the instance.